doc/classes/VSplitContainer.xml

```diff
@@ -22,5 +22,8 @@
 		<theme_item name="grabber" data_type="icon" type="Texture2D">
 			The icon used for the grabber drawn in the middle area.
 		</theme_item>
+		<theme_item name="split_bar_background" data_type="style" type="StyleBox">
+			Determines the background of the split bar if its width is greater than zero.
```

Comment:
```suggestion
			Determines the background of the split bar if its height is greater than zero.
```
The split bar is vertical in VSplitContainer.
Although maybe a more generic word would be better, like "thickness", so that it can be used in all classes.

Reply:
Will do.